drivers/sensor/melexis/Kconfig

```
@@ -0,0 +1,6 @@
+# Copyright (c) 2024 Florian Weber <Florian.Weber@live.de>
+# SPDX-License-Identifier: Apache-2.0
+
+# zephyr-keep-sorted-start
+source "drivers/sensor/melexis/MLX90394/Kconfig"
```

Convention is to use lower case for directory names
```suggestion
source "drivers/sensor/melexis/mlx90394/Kconfig"
```

---

done